.drone.yml

```diff
@@ -8,7 +8,7 @@ platform:
 
 steps:
 - name: test
-  image: golang:1.12
+  image: golang:1.16
```

Comment:
not sure if we should do this as part of this PR, is this something we should push to all of drone ?